common/src/main/java/com/microsoft/identity/common/adal/internal/net/DefaultConnectionService.java

```diff
@@ -50,7 +50,9 @@
 
     private final Context mConnectionContext;
     private static boolean connectionAvailable = false;
```

Comment:
If this is private, do we need to register the callback in the first place? Is it being used anywhere?

If not, it's probably safer to just removing it.

Reply:
Yes, it is used in the `isConnectionAvailable` method.

Reply:
The reason to handle this via a callback is that the other mechanism is deprecated and broken in newer versions of android.

Reply:
Got it. Thanks.